### DISCUSSION TITLE
Bytedance Applications

### DISCUSSION THREAD

**Discussion by AkshatMittal7** [ReaVNaiL/New-Grad-2024#204] [General]
I don't think you can apply to Bytedance new grad positions if you've already completed two applications for Tiktok new grad. Can someone please confirm this?

**Comment by likhitgarimella** [comment]
> I don't think you can apply to Bytedance new grad positions if you've already completed two applications for Tiktok new grad. Can someone please confirm this?

Yes that's correct. I noticed that too. Perhaps trying a different email id might work?

**Comment by nahianreza** [comment]
You can always make a new account

**Reply by bryanipablo** [reply]
I think they can disqualify for that. I remember seeing something about that on their site

**Comment by henryzhang03** [comment]
"Candidates can apply to a maximum of two positions and will be considered for jobs in the order you apply. The application limit is applicable to TikTok and its affiliates' jobs globally." ByteDance is TikTok's parent company, so I assume this statement holds true.
